thirdeye/thirdeye-frontend/app/pods/self-serve/create-alert/template.hbs

```diff
@@ -80,6 +69,17 @@
       {{/power-select}}
     </div>
 
+    <div class="col-xs-6">
```

Comment:
Switching "filters" and "dimensions" fields (dimension to the left)

Reply:
Did we check with design that this change is fine? (not sure if Selene had the previous order for a specific reason)

Reply:
will check